### PR TITLE
Update with recently created repos

### PR DESCRIPTION
**Pull request by erikbosch**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -101,6 +101,6 @@ Repositories in status Beta or Production may also occasionally be security audi
 
 |   Repository        | Status         | Content/Comment
 |-----------------------|----------------|---------------------|
-| [kuksa.val](https://github.com/eclipse/kuksa.val)| Alpha/Deprecated | Feeders/Providers
-| [kuksa.val.feeders](https://github.com/eclipse/kuksa.val.feeders)| Production/Mature/Deprecated | KUKSA Databroker, KUKSA Server,KUKSA Go Client and more
-| [kuksa.val.services](https://github.com/eclipse/kuksa.val.services)| Alpha | Examples services using KUKSA
+| [kuksa.val](https://github.com/eclipse/kuksa.feeders)| Production/Mature/Deprecated | KUKSA Databroker, KUKSA Server, KUKSA Go Client and more
```
Comment (SebastianSchildt) [inline profile/README.md:104]: I guess the link is wrong and shoudl go to kuksa.val instead?

File: profile/README.md [hunk 2]
```diff
@@ -58,7 +57,7 @@ Repositories in status Beta or Production may also occasionally be security audi
 | [kuksa-dds-provider](https://github.com/eclipse-kuksa/kuksa-dds-provider)| Beta | Python-based provider for DDS
 | [kuksa-gps-provider](https://github.com/eclipse-kuksa/kuksa-gps-provider)| Beta | Python-based provider for gpsd
 | [kuksa-someip-provider](https://github.com/eclipse-kuksa/kuksa-someip-provider)| Alpha | Python-based PoC provider for SOME/IP using vsomeip
-
+| [kuksa-mock-provider](https://github.com/eclipse-kuksa/kuksa-mock-provider)| Alpha | Python-based mock provider intended for testing
```
Comment (SebastianSchildt) [inline profile/README.md:60]: Only a nitpick, but "intended for testing" reads like the mock provider is intended to BE tested (also true, it beign alpha and all), but I think the intention was more something like

Python-based provider for mocking VSS signals (to help testing VSS applications)

**Review by SebastianSchildt — CHANGES_REQUESTED** [review]
I think one link has a typo, see comment

**Comment by erikbosch** [comment]
Updated according to latest comments

**Review by SebastianSchildt — APPROVED** [review]
Lgtm

**Pull request merged**